platforms/fabric/src/main/java/com/dfsek/terra/fabric/generation/FabricChunkGeneratorWrapper.java

```diff
@@ -209,6 +181,11 @@ public VerticalBlockSample getColumnSample(int x, int z, HeightLimitView height)
         return new VerticalBlockSample(height.getBottomY(), array);
     }
     
+    @Override
+    public void getDebugHudText(List<String> text, BlockPos pos) {
+    
+    }
+    
```

Comment:
This should probably display (among other things)
- structure
- biome
- noise value for terrain gen

Reply:
Biome is the only possible option, and it's already displayed elsewhere in the HUD